wrappers/opencv/face/readme.md

```diff
@@ -0,0 +1,13 @@
+# rs-imshow Sample
```

Comment:
Copy&pasted

Reply:
This file needs to be taken out of the changes -- the opencv/face/ example is not going in...